wrapper/src/main/java/software/amazon/jdbc/dialect/RdsPgDialect.java

```diff
@@ -34,7 +34,8 @@ public class RdsPgDialect extends PgDialect {
 
   private static final Logger LOGGER = Logger.getLogger(RdsPgDialect.class.getName());
 
-  private static final List<String> dialectUpdateCandidates = Collections.singletonList(
+  private static final List<String> dialectUpdateCandidates = Arrays.asList(
+      DialectCodes.RDS_MULTI_AZ_PG_CLUSTER,
```

Comment:
minor nit: will this list ever change? If not then can we use something like Collections.unmodifiableList?

Reply:
We can use `Collections.unmodifiableList` but this list is a private member and there's no chance that user, or a custom user plugin, can change it, intentionally or accidentally.